### PR TITLE
Added King Kong Movie in 2005 folder

### PR DESCRIPTION
**Pull request by Apoorv-Gaurav-Agarwal**

### CODE REVIEW

File: movies/2005/King Kong.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "King Kong",
+  "year": 2005,
+  "runtime": 201,
```
Comment (mikeleguedes) [inline movies/2005/King Kong.json]: 187

File: movies/2005/King Kong.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "categories": [
+    "drama",
+    "fantasy"
```
Comment (mikeleguedes) [inline movies/2005/King Kong.json]:
```suggestion
    "fantasy"
    "action"
    "adventure
    "romance"
```

Comment (mikeleguedes) [reply movies/2005/King Kong.json]: there are more categories that king kong is included

**Comment by Apoorv-Gaurav-Agarwal** [comment]
Edited the changes.

**Comment by Apoorv-Gaurav-Agarwal** [comment]
Any update on this?

**Comment by Apoorv-Gaurav-Agarwal** [comment]
@mikeleguedes Please approve the changes and merge the branch

**Pull request merged**